build-scripts/cassandra-dtest-pytest.sh

```diff
@@ -100,7 +100,7 @@ if [ "x${DTEST_SPLIT_CHUNK}" != "x" ] ; then
     SPLIT_TESTS=$(split -n r/${DTEST_SPLIT_CHUNK} ${WORKSPACE}/test_list.txt)
 fi
 
-PYTEST_OPTS="-vv --log-cli-level=DEBUG --junit-xml=nosetests.xml --junit-prefix=${DTEST_TARGET} -s"
+PYTEST_OPTS="-vv --durations=100 --log-cli-level=DEBUG --junit-xml=nosetests.xml --junit-prefix=${DTEST_TARGET} -s"
```

Comment:
awesome idea! 

note, this will be the slowest 100 in each split.

and… you can sort all tests by duration here: https://ci-cassandra.apache.org/job/Cassandra-4.1/lastSuccessfulBuild/testReport/